modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java

```diff
@@ -326,18 +334,23 @@ public List<ServiceTemplateEntity> listServiceTemplates(ServiceTemplateQueryMode
      */
     public void reviewServiceTemplateRegistration(UUID id, ReviewRegistrationRequest request) {
         ServiceTemplateEntity existingTemplate = getServiceTemplateDetails(id, false, true);
-        if (ServiceRegistrationState.APPROVED == existingTemplate.getServiceRegistrationState()
-                || ServiceRegistrationState.REJECTED
-                == existingTemplate.getServiceRegistrationState()) {
+        ServiceTemplateRegistrationState state =
```

Comment:
just to highlight.. that 'Review' must handle the update case differently.

Reply:
The update service not update yet. I think here is any need to modify now

Reply:
The service `updateServiceTemplate` not update yet. this will be updated in other issues.